rust/otap-dataflow/crates/config/src/pipeline/service/telemetry/metrics/views.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Metrics views level configurations.
+
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+/// Opentelemetry Metrics View configuration.
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct ViewConfig {
+    /// Selector to match instruments for this view.
+    pub selector: ViewSelector,
+
+    /// Stream configuration for this view.
+    pub stream: ViewStream,
+}
+
+/// Opentelemetry Metrics View Selector configuration.
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct ViewSelector {
+    /// The name of the instrument to match.
+    pub instrument_name: Option<String>,
+    // TODO: Add more selector fields.
+}
+
+/// Opentelemetry Metrics View Stream configuration.
+#[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
+pub struct ViewStream {
+    /// The name of the instrument.
+    pub name: Option<String>,
+    /// The description of the instrument.
+    pub description: Option<String>,
+}
```

Comment:
For people who are not fully familiar with the concepts of view, selector, and stream, it would be good to add some comments describing them in more detail.

nit: Opentelemetry -> OpenTelemetry